src/transactions/TransactionFrame.cpp

```diff
@@ -740,6 +740,19 @@ TransactionFrame::applyOperations(SignatureChecker& signatureChecker,
                                   TransactionMeta& outerMeta)
 {
     ZoneScoped;
+    auto& internalErrorCounter = app.getMetrics().NewCounter(
```

Comment:
Because previously it was being lazy-initialized. If no exception occurs, instead of outputting "internal-error" count = 0, it'll complain about "internal-error" not being a valid metric.